src/ert/_c_wrappers/enkf/config/enkf_config_node.py

```diff
@@ -202,47 +147,45 @@ def createSummaryConfigNode(
 
     @classmethod
     def createFieldConfigNode(
-        cls, key, grid: EclGrid, trans_table=None, forward_init=False
+        cls, key, grid: EclGrid, trans_table=None
     ) -> "EnkfConfigNode":
-        return cls._alloc_field_node(key, grid, trans_table, forward_init)
+        config_node = cls._alloc_field_node(key, grid, trans_table)
+        return config_node
 
     @classmethod
     def create_ext_param(
         cls,
         key: str,
         input_keys: Union[List[str], Dict[str, List[Tuple[str, str]]]],
-        output_file: Optional[str] = None,
     ) -> ExtParamConfig:
         config = ExtParamConfig(key, input_keys)
-        output_file = output_file or key + ".json"
+        # var_type EnkfVarType.EXT_PARAMETER
```

Comment:
comment is a bit terse, would prefer to have it be a bit more filled out.

Reply:
Removed.